opentracing-api/src/main/java/io/opentracing/References.java

```diff
@@ -29,6 +29,12 @@ private References(){}
     public static final String CHILD_OF = "child_of";
 
     /**
+     * Like {@link References#CHILD_OF}, but inferred via {@link Scheduler#activeContext}, not explicitly
+     * specified by the programmer (and thus less certain).
+     */
+    public static final String INFERRED_CHILD_OF = "inferred_child_of";
```

Comment:
@yurishkuro that's a great point – I created this constant early on but see no use case for it with the way this version comes together. (not to say that implicit/inferred children shouldn't be labelled as such somehow... but agree that it doesn't need to be in the public API)